orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/CreateBakeTask.groovy

```diff
@@ -99,12 +99,12 @@ class CreateBakeTask implements RetryableTask {
   }
 
   private static boolean shouldRebake(Stage stage) {
-    if (stage.context.rebake) {
+    if (stage.context.rebake == true) {
```

Comment:
Not null, but it could be anything other than "false" and still evaluate to `true` which isn't exactly ideal IMO.

Loose typing FTL.